### PR TITLE
Get document added in extract_czml

### PR DESCRIPTION
**Pull request by Syed-Osama-Hussain**

### CODE REVIEW

File: src/poliastro/czml/extract_czml.py [hunk 1]
```diff
@@ -493,3 +493,6 @@ def add_orbit(
             self.packets.append(pckt)
 
         self.i += 1
+
+    def getDocument(self):
```
Comment (astrojuanlu) [inline src/poliastro/czml/extract_czml.py]: `get_document`, please :innocent:

File: tests/test_czml.py [hunk 2]
```diff
@@ -2,18 +2,73 @@
...
+    start_epoch = iss.epoch
+    end_epoch = iss.epoch + molniya.period
+
+    sample_points = 10
+
+    pr_map_url = (
+        "https://upload.wikimedia.org/wikipedia/commons/c/c4/Earthmap1000x500compac.jpg"
+    )
+    scene = False
+    # expected_packet = """[
```
Comment (astrojuanlu) [inline tests/test_czml.py]: What's this long comment?

Comment (Syed-Osama-Hussain) [reply tests/test_czml.py]: Sorry I think I left it there while debugging.

File: tests/test_czml.py [hunk 3]
```diff
@@ -53,7 +74,7 @@ def test_czml_custom_packet():
     )
 
     pckt = extractor.packets[-1]
-
+
```
Comment (astrojuanlu) [inline tests/test_czml.py:77]: :eyes:

**Comment by Syed-Osama-Hussain** [comment]
Okay.

**Comment by astrojuanlu** [comment]
Thanks a lot @Syed-Osama-Hussain! :tada:

**Pull request merged**